src/gui.js

```diff
@@ -346,6 +346,23 @@ IDE_Morph.prototype.init = function (config) {
 IDE_Morph.prototype.openIn = function (world) {
     var hash, myself = this;
 
+    window.onmessage = function (event) {
+        // make the API accessible from outside an iframe
+        var ide = world.children[0];
```

Comment:
Maybe you just forgot to push something, but doesn't this still use the global? 🙃 


```suggestion
        var ide = myself;
```

Reply:
haha, well, this `world` is the parameter you're passing to `openIn`, not a global var, but you're still right that `myself` is the right thing to do :)